source/tomopy/misc/corr.py

```diff
@@ -557,7 +557,7 @@ def remove_ring(rec, center_x=None, center_y=None, thresh=300.0,
     thresh_min : float, optional
         min value for portion of image to filer
     theta_min : int, optional
-        minimum angle in degrees (int) to be considered ring artifact
+        minimum half angle in degrees (int) to be considered ring artifact
```

Comment:
Is "half angle" a common phrase? It is not immediately obvious to me that this means we multiply this parameter by two to get the minimum ring artifact size.
```suggestion
        one half of the minimum angle (degrees) to be considered a ring artifact
```
```suggestion
        features larger than twice this angle (degrees) will be considered a ring artifact
```

Reply:
Not sure if its common but "half angle" is used sometimes. But I'd be happy with any of these suggestions.

Another option would be to divide the angle by 2 inside the function, so that it is the "minimum angle". But that would change the existing behaviour.